### PR TITLE
Create ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre-c8y

Ticket: [MTM-61669](https://cumulocity.atlassian.net/browse/MTM-61669)
Original PR: [7736](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7736)

[MTM-61669]: https://cumulocity.atlassian.net/browse/MTM-61669?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Enhance Dashboard Template Assignment for Untyped Devices  [GRAFT][release/cd] (#7736)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: The Claude AI suggested to use the following title:
```suggestion
title: Enhance dashboard template assignment for untyped devices
```

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61669
+version: 1021.15.0
+---
+Enhance Dashboard Template Assignment for Untyped Devices  [GRAFT][release/cd] (#7736)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, dashboard templates could not be assigned to devices without a device type, which limited the ability to monitor these devices effectively. With this change, it is now possible to assign dashboard templates to devices that have no device type specified. This allows users to create dashboards for any device, regardless of whether it has a type assigned, improving the flexibility and coverage of device monitoring.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]:
```suggestion
Previously, when device had no type property assigned, Dashboard template section in dashboard settings was hidden. Now, in this situation, section will be shown but user will be informed that enabling dashboard template will be possible only when type is added to device. To make it easier, there is a button that will navigate user to Device management app device view.
Changes also includes fix for child devices issue- before new approach to dashboard templates has been implemented, it was possible to add type dashboard to child device that had no c8y_IsDevice property. It is now possible again and c8y_IsDevice property is not necessary.
```

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61669
+version: 1021.15.0
+---
+Previously, when device had no type property assigned, Dashboard template section in dashboard settings was hidden. Now, in this situation, section will be shown but user will be informed that enabling dashboard template will be possible only when type is added to device. To make it easier, there is a button that will navigate user to Device management app device view.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]:
```suggestion
Previously, when a device had no type property assigned, the dashboard template section in the dashboard settings was hidden. Now, in this case, the section is visible but information is provided that the dashboard template is only enabled when a type is added to the device. Moreover a button is provided which navigates the user to the device view in the Device Management application.
```

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+Changes also includes fix for child devices issue- before new approach to dashboard templates has been implemented, it was possible to add type dashboard to child device that had no c8y_IsDevice property. It is now possible again and c8y_IsDevice property is not necessary.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: I think we have an extra ticket/entry for this change?

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
+title: Enhance dashboard template assignment for untyped devices
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]:
```suggestion
title: Dashboard template now visible for devices without type property
```

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md (hunk 5 above)
Comment (jdre-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: for the fix? we do https://cumulocity.atlassian.net/browse/MTM-61830

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md (hunk 6 above)
Comment (jdre-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: that would suggest that existing dashboard instance with assgined type (dashboard template) is now visible for devices without type property. That's not true- main change of this PR is that assigning type to dashboard (making them dashboard template) is now enhanced for devices without property

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md (hunk 5 above)
Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: @jdre-c8y Then we should remove the last section from here.

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md (hunk 6 above)
Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: @jdre-c8y Please suggest a correct user-friendly phrasing

Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: actually this one small change in your suggestion should do the work
```suggestion
title: Dashboard template setting now visible for devices without type property
```

Comment (jdre-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]: @BeateRixen actually this one small change in your suggestion should do the work
```suggestion
title: Dashboard template setting now visible for devices without type property
```

File: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61669
+version: 1021.15.0
+---
+Previously, when a device had no type property assigned, the dashboard template section in the dashboard settings was hidden. Now, in this case, the section is visible but information is provided that the dashboard template is only enabled when a type is added to the device. Moreover a button is provided which navigates the user to the device view in the Device Management application.
+Changes also includes fix for child devices issue- before new approach to dashboard templates has been implemented, it was possible to add type dashboard to child device that had no c8y_IsDevice property. It is now possible again and c8y_IsDevice property is not necessary.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md]:
```suggestion
```

**Review by BeateRixen — APPROVED** [review]

**Review by luja-c8y — APPROVED** [review]

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**